ow_plexil/src/plans/FaultHandlingPattern4.plp

```diff
@@ -0,0 +1,142 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Pattern summary: Execute a sequence of actions. If a fault is injected
+// during a related action, the plan will stop the action. If/when the fault
+// is cleared, the action will automatically be attempted again.
```

Comment:
I suggest adding a note that if you inject an arm fault during an arm operation, you should wait until you see the action aborted, e.g. "Stow finished in state ABORTED" before clearing the fault, otherwise the action will show as completed successfully (even though the arm halted mid-way) and will not be re-attempted.  A combination of bugs in the simulator and unavailability of ROS action state in PLEXIL are the culprit here.

Reply:
Do you think the comments in UnstowArm and StowArm are too long? It seems this statement you are talking about adding (which I think is very important) is related.

Reply:
Yes, it's related.  I'll leave it to you to figure where to best place the comments.  You could probably avoid the repetition in Unstow() and Stow() that is there now.